### ISSUE TITLE
Slow sitemap index

### ISSUE THREAD

**Issue by wpsmort** [semperfiwebdesign/all-in-one-seo-pack#2648]
Reported by Troy by email today:

> I have installed v3.1 and I can't update or create a sitemap. The plugin keeps timing out. I have contacted my host and they can't figure it out either.

We cloned the customer site and verified that it was taking over 20 seconds to display the sitemap index file.  The index contains 10 sitemaps for pages, products and product categories.  The site has 0 posts, 839 pages, 6379 products and 536 product categories.  Images are excluded from the sitemap.  Pagination is set to 1000.

Rolling back to v2.12 and v2.113 of Pro works fine.

We need to determine why the index takes so long to load and appears to be using around 7MB of memory.

**Comment by michaeltorbert** [comment]
I'd like to see this in a 3.1.1 release.

**Comment by wpsmort** [comment]
After further testing on multiple clones of the customer site, we found that we could not reproduce the problem when switching between versions from 3.0 to 3.1.  Each time we activated a different version the problem went away.  We testing on the original clone and couldn't reproduce it there.  

Testing on the live customer site we found that the site ran slow with AIO deactivated.  The customer confirmed he was aware of performance problems and was moving to a different host.
